apps/common/receipt.py

```diff
@@ -103,19 +104,35 @@ def make_qrfile(data, **kwargs):
     return qrfile
 
 
-def format_inline_qr(data):
-    qrfile = make_qrfile(data, kind="svg", svgclass=None)
+def make_epc_qrfile(payment, **kwargs):
+    qrfile = io.BytesIO()
+    qr = helpers.make_epc_qr(
+        name="Electromagnetic Field Ltd",
+        iban="GB21BARC20716472954433",
+        amount=payment.amount,
+        reference=payment.bankref,
+        encoding=1,
+    )
+    qr.save(qrfile, **kwargs)
+    qrfile.seek(0)
+    return qrfile
 
-    root = etree.XML(qrfile.read())
-    # Allow us to scale it with CSS
-    root.attrib["viewBox"] = "0 0 %s %s" % (root.attrib["width"], root.attrib["height"])
-    del root.attrib["width"]
-    del root.attrib["height"]
-    root.attrib["preserveAspectRatio"] = "none"
 
+def qrfile_to_svg(qrfile):
+    root = etree.XML(qrfile.read())
```

Comment:
I guess the XML parsing is unecessary, this should work as well:

```python
return Markup(qrfile.getvalue().decode('utf-8'))
```
If you make the detour via lxml to get rid of the XML declaration, you can also tell Segno to omit it (line 127 and 132):

```python
# Omit XML declaration and trailing new line 
qrfile = make_qrfile(data, kind="svg", svgclass=None, omitsize=True, xmldecl=False, nl=False)
```

If the QR code is used in a HTML5 context, the SVG namespace declaration may be superfluous as well:
```python
# Omit XML declaration, SVG namespace and trailing new line 
qrfile = make_qrfile(data, kind="svg", svgclass=None, omitsize=True, xmldecl=False, svgns=False, nl=False)
```

Reply:
Neat; taking a cautious approach here before dropping the XML and namespace declarations, but removing the XML parsing certainly seems a safe & nice improvement to start with.

Reply:
No problem, but I guess dropping the XML declaration is safe because
```
etree.tostring(root).decode("utf-8")
```
returned the XML markup without an XML declaration anyway.